frontend/views/pages/GroupSettings.vue

```diff
@@ -80,6 +80,8 @@ page.c-page
           data-test='saveBtn'
         ) {{ L('Save changes') }}
 
+  roles-and-permissions
+
```

Comment:
Can we have this section displayed only in development mode since it doesn't work and hide it in production?

Reply:
Sure, updated in [here](https://github.com/okTurtles/group-income/pull/2507/commits/4eaec1e0432e4c0a3dec6b90f3d2b8cb7dada70a)